tests/typing/mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy]
```

Comment:
This file added for mypy tests because the pytest plugin runs these tests in a subprocess, where the relative path to the plugin in the main mypy.ini no longer points to the plugin